test/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jboss.mod_cluster</groupId>
         <artifactId>mod_cluster-parent</artifactId>
-        <version>2.0.5.Final-SNAPSHOT</version>
+        <version>2.1.0.Final-SNAPSHOT</version>
```

Comment:
We will need to figure this out at some point :-) but at least we are testing the latest and greatest.